synapse/module_api/__init__.py

```diff
@@ -1230,6 +1231,42 @@ def looping_background_call(
                 f,
             )
 
+    def delayed_background_call(
+        self,
+        msec: float,
+        f: Callable,
+        *args: object,
+        desc: Optional[str] = None,
+        **kwargs: object,
+    ) -> IDelayedCall:
+        """Wraps a function as a background process and calls it in a given number of milliseconds.
```

Comment:
Is it worth mentioning that if the service is restarted then the task won't run?